app/src/main/res/layout/generated_test_locationcomponent.xml

```diff
@@ -12,6 +12,8 @@
     mapbox:mapbox_locationComponentAccuracyRingBorderColor = "#000000"
     mapbox:mapbox_locationComponentLayerAbove = "testString"
     mapbox:mapbox_locationComponentLayerBelow = "testString"
+    mapbox:mapbox_locationComponentPuckBearingEnabled = "false"
```

Comment:
Why is it disabled?

Reply:
This is a generated test value, we set to false for all boolean values